src/sentry/integrations/metric_alerts.py

```diff
@@ -96,3 +105,132 @@ def incident_attachment_info(incident, new_status: IncidentStatus, metric_value=
         "ts": ts,
         "title_link": title_link,
     }
+
+
+def metric_alert_attachment_info(
+    alert_rule: AlertRule,
+    selected_incident: Optional[Incident] = None,
+    new_status: IncidentStatus = None,
+    metric_value=None,
+    unfurl=False,
+):
+    logo_url = absolute_uri(get_asset_url("sentry", "images/sentry-email-avatar.png"))
+
+    latest_incident = None
+    if selected_incident is None:
+        latest_incident = Incident.objects.filter(
+            id__in=Incident.objects.filter(alert_rule=alert_rule)
+            .values("alert_rule_id")
+            .annotate(incident_id=Max("id"))
+            .values("incident_id")
+        ).first()
+
+    status = INCIDENT_STATUS[IncidentStatus.CLOSED]
+    if new_status:
+        status = INCIDENT_STATUS[new_status]
+    else:
+        if selected_incident:
+            status = INCIDENT_STATUS[IncidentStatus(selected_incident.status)]
+        elif latest_incident:
+            status = INCIDENT_STATUS[IncidentStatus(latest_incident.status)]
+
+    title = f"{status}: {alert_rule.name}"
+
+    if unfurl:
```

Comment:
I think both the `if` and `else` generate the same `title_link` here. I think the `reverse()` version is preferred.

Reply:
And if so, then the `unfurl` parameter would be unused.